client/my-sites/plugins/plans/index.tsx

```diff
@@ -51,6 +71,19 @@ const Plans = () => {
 				subHeaderText={ `Choose the plan that's right for you and reimagine what's possible with plugins` }
 				brandFont
 			/>
+			<div className="plans">
+				<PlansFeaturesMain
+					site={ selectedSite }
+					isInSignup={ false }
+					isInMarketplace
+					intervalType={ intervalType }
+					onUpgradeClick={ onSelectPlan }
+					flowName={ 'marketplace' }
+					shouldShowPlansFeatureComparison
+					showFAQ={ false }
```

Comment:
Dead code. Issue here https://github.com/Automattic/wp-calypso/issues/68003

Reply:
Oops, I made it look confusing here.
It's part of this PR: https://github.com/Automattic/wp-calypso/pull/67891/files#diff-2545c62593dd002397248377b1c0feb643bd2acc7694054ce094997b1c029aceR460-R463

Reply:
I created a new PR for it: https://github.com/Automattic/wp-calypso/pull/68010

Reply:
Just noticing you already fixed this [here](https://github.com/Automattic/wp-calypso/pull/67891/files?w=1#diff-2545c62593dd002397248377b1c0feb643bd2acc7694054ce094997b1c029aceR461-R462). :face_exhaling: